liana-gui/src/installer/view/mod.rs

```diff
@@ -1150,7 +1159,8 @@ pub fn define_electrum<'a>(address: &form::Value<String>) -> Element<'a, Message
             .size(text::P1_SIZE)
             .padding(10),
         )
-        .push(text(electrum::ADDRESS_NOTES).size(text::P2_SIZE))
```

Comment:
nit: Did you intend to remove the `.size(text::P2_SIZE)`?

Reply:
> yes I think I removed intentionally the size to stay consistant in the context, IIRC i notice the size was different from some others warnings